### ISSUE TITLE
Startup order integration / add-on

### ISSUE THREAD

**Issue by IngmarStein** [savdagod/TidbytAssistant#15]
After I start the box Home Assistant is running on, the TidbytAssistant integration usually fails to initialize because the add-on is not yet started:

```
2024-11-26 20:19:22.678 ERROR (SyncWorker_9) [custom_components.tidbytassistant] Could not connect to the add-on. Make sure it is installed or running and try again.
2024-11-26 20:19:22.678 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'tidbytassistant': Integration failed to initialize.
```

This also fills the logs with errors from all automations using TidbytAssistant.

Restarting Home Assistant and closing all errors resolves the issue.

If there is no way to delay the initialization of the integration until the add-on is ready, maybe it can retry multiple times?

**Comment by savdagod** [comment]
This is on my list to fix as I’ve also encountered this a few times when restarting my machine.

**Comment by paxos** [comment]
Seeing this as well. Have not been able to run the integration so far.
Thank you a lot for this project :)

**Comment by paxos** [comment]
Got it working by downgrading to v1.0.12 - otherwise i see the same error as @IngmarStein

**Comment by savdagod** [comment]
Fixed in 1.0.14
